srcpkgs/resynthesizer/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'resynthesizer'
+pkgname=resynthesizer
+version=2.0.3
+revision=1
+wrksrc="${pkgname}-${version}"
```

Comment:
This is the default value of `wrksrc` you don't have to add it manually.

Reply:
Gotcha, I'll change it. Is there a way to update the PR without causing a rebuild?